projects/fizzbuzz/python/Lightner/README.md

```diff
@@ -0,0 +1,5 @@
+To run FizzBuzz program with Python
+
+* Open Python
```

Comment:
calling `python FizzBuzz.py 17` from the terminal is clearer

not sure wht you mean by open python